code/modules/medical/diseases/braincrit.dm

```diff
@@ -0,0 +1,188 @@
+	reagentcure = list( "coffee" = list(1,10),	// Sort of a stimulant
+						"espresso" = 1,			// Even slightly more of a stimulant
+						"saline" = 2,			// 
+						"mannitol" = 5, 		// Brain meds
+						"epinephrine" = 5, 		// 
+						"synaptizine" = 15)) 	// i like synaptizine :D
+	recureprob = 10
+	affected_species = list("Human","Monkey")
+	stage_prob = 5
+
+/datum/ailment/malady/brainfailure/stage_act(var/mob/living/affected_mob,var/datum/ailment_data/malady/D)
+	if (..())
+		return
+
+	if (ishuman(affected_mob))
+		var/mob/living/carbon/human/H = affected_mob
+		if (!H.organHolder)
+			H.cure_disease(D)
+			return
+		if (!H.organHolder.brain)
+			H.cure_disease(D)
+			return
+
+	switch (D.stage)
+		if (1)
+			if (prob(1) && prob(10))
+				boutput(affected_mob, "<span class='notice'>Your head feels somewhat less painful.</span>")
+				affected_mob.cure_disease(D)
+				return
+			if (prob(5))
+				affected_mob.changeStatus("slowed", rand(10,20))
+				affected_mob.change_misstep_chance(3)
+				boutput(affected_mob, "<span class='alert'>You suddenly find yourself struggling to keep balance.</span>")
+			if (prob(8))
+				affected_mob.emote(pick("pale", "shudder"))
+			if (prob(5))
+				boutput(affected_mob, "<span class='alert'>A clear, slippery fluid drips from your nose.</span>")
+				affected_mob.take_brain_damage(1)
+			if (prob(5))
+				boutput(affected_mob, "<span class='alert'>Your [pick("nose starts", "ears start"] bleeding.</span>")
+				affected_mob.take_bleeding_damage(affected_mob, null, 0, DAMAGE_STAB, 1)
+			else if(prob(12))
+				affected_mob.stuttering = max(10, affected_mob.stuttering)
+				boutput(affected_mob, "<span class='alert'>You feel [pick("numb", "confused", "dizzy", "dazed")].</span>")
+				affected_mob.emote("collapse")
+
+		if (2)
+			if (prob(8))
+				boutput(affected_mob, "<span class='alert'>Your head [pick("feels like shit","hurts like fuck","pounds horribly","twinges with an awful pain")].</span>")
+				affected_mob.emote(pick("groan", "grimace"))
+				affected_mob.change_misstep_chance(3)
+			if (prob(5))
+				affected_mob.changeStatus("slowed", rand(30,45))
+				affected_mob.change_misstep_chance(4)
+				boutput(affected_mob, "<span class='alert'>Your legs feel [pick("oddly ","")]weak.</span>")
+			if (prob(5))
+				boutput(affected_mob, "<span class='alert'>A clear, slippery fluid seeps from your nose.</span>")
+				affected_mob.take_brain_damage(1)
+			if (prob(5))
+				boutput(affected_mob, "<span class='alert'>You sneeze, spraying blood everywhere!</span>")
+				affected_mob.bleed(affected_mob, 2, 2)
+			if (prob(8))
+				affected_mob.emote("gasp")
+				affected_mob.losebreath+=3
+			if (prob(12))
+				boutput(affected_mob, "<span class='alert'>You feel [pick("numb", "confused", "dizzy", "dazed")].</span>")
+				affected_mob.drowsyness += 2
+			if (prob(1) && prob(1) && prob(1))	// Thanks, find+replace!
+				boutput(affected_mob, "<span class='alert'>Your brain stops beating!</span>")
```

Comment:
I can't tell if this is intentional, but it's funny.^^

Reply:
=3